bookbotproject/conversationtree/migrations/0022_auto_20171206_2247.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11 on 2017-12-06 19:47
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
```

Comment:
Please delete empty lines at lines 4, 6, 9, 13. Everything else looks fine. :)

Reply:
Since this is auto generated file, I don't think we should spend time on the redundacies or non erroneous code fragments as these lines. Actually I find it a little dangerous also, since they are generated, changing them could harm the way system working. Do you see possible threats to the project related with these lines stays put?

Reply:
Yes, I didnt realize it was a migration file, sorry :)